models/constants.go

```diff
@@ -46,6 +46,9 @@ const SegmentHMSet = "Redis/HMSet"
 //SegmentZRangeBy represents a segment
 const SegmentZRangeBy = "Redis/ZRangeBy"
 
+//SegmentSMembers represents a segment
```

Comment:
Why do you need this constant?

Reply:
For tracing. But I wasn't using though. I updated the PR, and now I am:
```
err := mr.WithSegment(SegmentSMembers, func() error {
		var err error
		redisKeys, err = redisClient.SMembers(
			GetRoomStatusSetRedisKey(schedulerName, StatusTerminating),
		).Result()
		return err
	})
```